src/address.d.ts

```diff
@@ -17,6 +17,7 @@ export interface Blech32Result {
 export interface ConfidentialResult {
     blindingKey: Buffer;
     unconfidentialAddress: string;
+    scriptPubKey?: Buffer;
```

Comment:
why optional?

Reply:
I was thinking that since it exported some other code could use this type and will break compatibility. But not sure